src/Classes/ServiceAPI/MyRadio_Track.php

```diff
@@ -532,11 +532,12 @@ public function setDigitisedBy($digitisedby)
      */
     public function setLastEdited()
     {
-        $this->lastedited = time();
+        $this->last_edited_time = time();
+        $this->last_edited_memberid = $_SESSION['memberid'];
         self::$db->query(
-            'UPDATE rec_track SET lastedited=$1, lasteditedby=$2 WHERE trackid=$3',
+            'UPDATE rec_track SET last_edited_time=$1, last_edited_memberid=$2 WHERE trackid=$3',
             [
-                $this->lastedited ? CoreUtils::getTimestamp($this->lastedited) : null,
+                $this->last_edited_time ? CoreUtils::getTimestamp($this->last_edited_time) : null,
```

Comment:
this ternary is still unnecessary, as you always set the variable. Not the conversion, but the `foo ? foo : bar`